code/parse/sexp.cpp

```diff
@@ -40631,8 +40648,19 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t1: Time in seconds that the supernova lasts.  Note that it will actually hit the player at " SCP_TOKEN_TO_STR(SUPERNOVA_HIT_TIME) " seconds.  If you want the HUD gauge to adjust for this, use the '$Supernova hits at zero' game_settings.tbl option.\r\n"},
 
 	{ OP_SUPERNOVA_STOP, "supernova-stop\r\n"
-		"\t Stops a supernova in progress.\r\n"
-		"\t Note this only works if the camera hasn't cut to the player's death animation yet.\r\n"},
+		"\tStops a supernova in progress.  Note this only works if the camera hasn't cut to the player's death animation yet.\r\n"},
```

Comment:
Very minor, but might be worth mentioning what stage that is, since a fredder now has access to it (3?).

Reply:
Ah, good call.